keep/api/core/db.py

```diff
@@ -141,7 +142,7 @@ def create_db_and_tables():
             logger.info("Migrating WorkflowToAlertExecution table")
             # get the foreign key constraint name
             results = session.exec(
```

Comment:
This is kinda shitty way to do migrations but we might have to add the column you added here like we do for other columns.
So try to add it and if you fail because it already exists just pass.

Reply:
sure thing and I agree on that one other think is it also fails on sqllite since sqllite doesnt have `information_schema`

Reply:
done